### PR TITLE
Make ScrollableTabBar more Android-like

### PR DESCRIPTION
**Pull request by dieppe**
- Center text vertically in tab
- Center active tab horizontally in container when possible

Also, allow for multiple tabs with the same label.

Tested on iOS and Android.

### CODE REVIEW

**Comment by skv-headless** [comment]
cool 👍 
Could you please make video or gif before and after?

**Comment by dieppe** [comment]
Here you go :)

Before:

scrollabletabbarold

After:

scrollabletabbarnew

File: ScrollableTabBar.js [hunk 1]
```diff
@@ -120,7 +126,7 @@ const ScrollableTabBar = React.createClass({
     const fontWeight = isTabActive ? 'bold' : 'normal';
 
     return <TouchableOpacity
-      key={name}
+      key={`${name}_${page}`}
```
Comment (skv-headless) [inline ScrollableTabBar.js]: Just interesting why do you need this?

File: ScrollableTabBar.js [hunk 2]
```diff
@@ -167,7 +173,6 @@ const ScrollableTabBar = React.createClass({
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         directionalLockEnabled={true}
-        scrollEventThrottle={16}
```
Comment (skv-headless) [inline ScrollableTabBar.js]: Just interesting why do you need this?

Comment (skv-headless) [inline ScrollableTabBar.js]: cool 👍

**Comment by skv-headless** [comment]
Just awesome 😍

File: ScrollableTabBar.js (hunk 1 above)
Comment (dieppe) [inline ScrollableTabBar.js]: This is totally unrelated to the android-like behavior, but I need to be able to have two tabs with the same name. 

I should remove the change from here and make another PR for that.

**Comment by Kielan** [comment]
is any example of this provided in the examples?

**Pull request merged**